client/src/components/Explorer/Explorer.tsx

```diff
@@ -1,12 +1,24 @@
-import PropTypes from 'prop-types';
+/* eslint-disable react/prop-types */
+
```

Comment:
Will prop warnings still trigger? Just checking we have a way for these components to be used outside of typescript land (ie. the ones added to global) and still have the benefit of prop warnings.

Reply:
You won't get runtime warnings anymore for components that have been switched to TypeScript and have had prop types removed.

It looks like it's possible to derive propt ypes from TypeScript interfaces using a plugin: https://github.com/milesj/babel-plugin-typescript-to-proptypes but it might be simpler to just switch everything to TypeScript?

Reply:
ok, we might need to work out what we want to do for components that are shared at some point, all good on these ones though.